_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,153 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each target element:
+
+- is categorized to follow the [flow content model][], and;
+- that it's contents are its [descendants][] in the DOM, and;
+- has at least one node in its contents that is a [palpable content][], which does not have the `hidden` attribute specified.
+
```

Comment:
It's weird that the expectation does not actually refer to https://html.spec.whatwg.org/multipage/dom.html#concept-element-content-model given that this is what the rule is supposed to test.